frame/support/src/storage/mod.rs

```diff
@@ -1335,6 +1337,9 @@ mod private {
 impl<T: Encode> StorageAppend<T> for Vec<T> {}
 impl<T: Encode> StorageDecodeLength for Vec<T> {}
 
+impl<T: Encode> StorageAppend<T> for BTreeSet<T> {}
+impl<T: Encode> StorageDecodeLength for BTreeSet<T> {}
+
```

Comment:
Could you please add a test that the `decode_len` and `append` functions exist and work correctly?

Reply:
I have gone ahead to include the tests @ggwpez